ios/brave-ios/scripts/scheme_preaction.py

```diff
@@ -43,8 +43,8 @@ def main():
     target_environment = 'simulator' if (options.platform_name
                                          == 'iphonesimulator') else None
 
-    UpdateSymlink(config, target_arch, target_environment)
     BuildCore(config, target_arch, target_environment)
+    UpdateSymlink(config, target_arch, target_environment)
```

Comment:
actually I think this call should be the last here, i.e. after `CallNpm(['npm', 'run', 'ios_pack_js'])`.